src/Components/Alert/Components/Modal/ModalHeader.tsx

```diff
@@ -18,5 +18,10 @@ export const ModalHeader: FC = () => {
       return <FiltersHeader />
     case "ALERT_CONFIRMATION":
       return <ConfirmationHeader />
+    case "ALERT_ARTWORKS":
+      return <Text variant="lg">View Artworks</Text>
+    default:
+      console.warn("no matching case is found in AlertContext")
```

Comment:
**question**: just wondering if such console warnings are useful in general and in this particular case, for debugging purposes maybe?

Reply:
It's only for debugging purposes
I added it because at some point the app was breaking while coding because the argument didn't match any of the values in switch-case and the default value is not specified

Do you think we could use something more practical